features/unattached_status.feature

```diff
@@ -123,3 +123,42 @@ Feature: Unattached status
            | focal   | yes      | no     | yes | yes  | yes         | yes   | no  | yes       |
            | hirsute | no       | no     | no  | no   | no          | no    | no  | no        |
            | impish  | no       | no     | no  | no   | no          | no    | no  | no        |
+
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    @uses.config.contract_token
+    Scenario Outline: Simulate status in a ubuntu machine
```

Comment:
Should we also have a test for the json output? Maybe to ensure it is the same schema as normal status output?

Reply:
hhhm yes - it will be a subset of the normal status output, but it's worth testing it. will add.